src/Microsoft.Identity.Web/AuthorityHelpers.cs

```diff
@@ -51,5 +53,23 @@ internal static string EnsureAuthorityIsV2(string authority)
             preserveAuthority = true;
             return authority;
         }
+         
+        internal static void AddAuthorityQueryToOptions(MicrosoftIdentityOptions options)
+        {
+            if (options.Authority != null)
+            {
+                int queryIndex = options.Authority.IndexOf('?', StringComparison.Ordinal);
+                if (queryIndex > -1)
+                {
+                    options.ExtraQueryParameters = options.ExtraQueryParameters ?? new Dictionary<string, string>();
```

Comment:
??= would clean this up

https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/operators/null-coalescing-operator

Reply:
Done, thank you